rmgpy/data/kinetics/kineticsTest.py

```diff
@@ -548,15 +545,30 @@ def testfilterReactions(self):
         
         reactants = np.unique(np.array(reactants)).tolist()
         products = np.unique(np.array(products)).tolist()
+        mreactants = np.array([i.molecule[0] for i in reactants])
+        mproducts = np.array([i.molecule[0] for i in products])
```

Comment:
Not particularly, sorry, that was leftover from the original test.  I've reformulated it now to work mostly with sets and use lists as needed.